src/libguac/guacamole/user-fntypes.h

```diff
@@ -116,6 +116,29 @@ typedef int guac_user_mouse_handler(guac_user* user, int x, int y,
 typedef int guac_user_key_handler(guac_user* user, int keysym, int pressed);
 
 /**
+ * Handler for Guacamole audio streams received from a user. Each such audio
+ * stream begins when the user sends a "audio" instruction. To handle received
```

Comment:
Yes: http://guac-dev.org/doc/gug/protocol-reference.html#audio-stream-instruction

The "audio" instruction was previously only used for server -> client. These changes allow it in either direction.